book/website/collaboration/inclusive-events/inclusive-events-checklist.md

```diff
@@ -0,0 +1,60 @@
+(cl-inclusive-events-checklist)=
+# Checklist
+
+> List out the action points you want the readers to take based on the key ideas in your chapter.
+> This list will be sectioned by event category
+
+
+(cl-inclusive-events-checklist-focus-session)=
+## Planning Inclusive Focus Sessions
+
+- Action point 1
+- Action point 2
+- Action point 3
+
+(cl-inclusive-events-checklist-interest-group)=
+## Planning Inclusive Community Sessions and Interest Group Events
+
+- Action point 1
+- Action point 2
+- Action point 3
+
+(cl-inclusive-events-checklist-seminar)=
+## Planning Inclusive Seminars, Lectures, and Fireside Chats
+
+- Action point 1
+- Action point 2
+- Action point 3
+
+(cl-inclusive-events-checklist-workshop)=
+## Planning Inclusive Workshops
+
+- Action point 1
+- Action point 2
+- Action point 3
+
+(cl-inclusive-events-checklist-conference)=
+## Planning Inclusive Conferences
+
+- Action point 1
+- Action point 2
+- Action point 3
+
+
```

Comment:
Perhaps add "Planning Inclusive Unconferences"?

Reply:
This is a great suggestion. 

Do you think we should add a different section specifically addressing unconferences to the chapter?

Reply:
Yes a separate/different sub-section before or after "Planning Inclusive Conferences" should be nice.